fullstack-cert/html-projects/framed-videos/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css"/> 
+  <title>Display Videos in an iframe</title>
+</head>
+<body>
+  <h1>How to display videos in an iframe</h1>
+  <br>
+  <iframe 
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/I0_951_MPE0"
+  align="middle" 
+  title="How do you embed videos onto your page using the iframe element?"
+  frameborder="0" 
+   allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+  referrerpolicy="strict-origin-when-cross-origin" 
+  allowfullscreen>
+  </iframe>
+  <main>
+    <h3>Lorem  Ipsum</h3>
+    <br/>
+    <div>
+      <p>Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>
+      <p>Consectetur adipiscing elit quisque faucibus ex sapien vitae.</p>
+      <p>Ex sapien vitae pellentesque sem placerat in id.  </p>
+      <p>Placerat in id cursus mi pretium tellus duis.</p>
+      <p>Pretium tellus duis convallis tempus leo eu aenean.</p>
+    </div>
+  </main>
```

Comment:
we can get rid of this lorem ipsum text and extra elements here.
the main goal of this workshop is to provide campers with some introductory practice to working with the iframe element.

so if the prototype only contains the h1 and iframe element itself, then that would suffice 👍🏾